fastly/block_fastly_service_v1_package.go

```diff
@@ -76,7 +76,11 @@ func (h *PackageServiceAttributeHandler) Read(ctx context.Context, d *schema.Res
 		ServiceVersion: s.ActiveVersion.Number,
 	})
 
-	if err != nil {
+	if err, ok := err.(*gofastly.HTTPError); ok && err.IsNotFound() {
+		log.Printf("[WARN] No wasm Package found for (%s), version (%v): %v", d.Id(), s.ActiveVersion.Number, err)
+		d.Set(h.GetKey(), nil)
+		return nil
+	} else if err != nil {
 		return fmt.Errorf("[ERR] Error looking up Package for (%s), version (%v): %v", d.Id(), s.ActiveVersion.Number, err)
 	}
```

Comment:
Wondering if we could refactor out the `else if` like so...

```suggestion
        if err != nil {
                if err, ok := err.(*gofastly.HTTPError); ok && err.IsNotFound() {
		        log.Printf("[WARN] No wasm Package found for (%s), version (%v): %v", d.Id(), s.ActiveVersion.Number, err)
		        d.Set(h.GetKey(), nil)
				        return nil
                }
                return fmt.Errorf("[ERR] Error looking up Package for (%s), version (%v): %v", d.Id(), s.ActiveVersion.Number, err)
        }
```